packages/opentelemetry-sdk-trace-base/src/Span.ts

```diff
@@ -364,8 +364,10 @@ export class Span implements APISpan, ReadableSpan {
   private _isSpanEnded(): boolean {
     if (this._ended) {
       diag.warn(
-        `Can not execute the operation on ended Span {traceId: ${this._spanContext.traceId}, spanId: ${this._spanContext.spanId}}`
+        `Cannot execute the operation on ended Span {traceId: ${this._spanContext.traceId}, spanId: ${this._spanContext.spanId}}. Change log level to debug for stack trace.`
       );
+      // Adding a debug log to capture the stack trace
+      diag.debug(`Stack trace for ended span operation: ${new Error().stack}`);
```

Comment:
Using just the stack trace string leads to dev tools not being able to apply custom formatting it does to error object - most notably the ability to jump to source code, which becomes inconsistent

```js logger_test.js
// Test 1
console.log(`stack test - inline - ${new Error().stack}`);

// Test 2
console.log('stack test - arg2', new Error().stack);

// Test 3
console.log('stack test - error obj', new Error());
```

In chrome, the experience is okay as chrome auto-linkifys links
![image](https://github.com/user-attachments/assets/2d442d47-03d7-4462-944e-3eb17043e023)

... as long as you don't expect to use sourcemaps - you just get lead into the built file instead of the source
![image](https://github.com/user-attachments/assets/799e1937-c6d3-4c75-ab23-0b363e29ba26)
![image](https://github.com/user-attachments/assets/57a81561-2bc4-4fc0-8938-b5b51a997719)

(realistically at otel bundle size, yeah I'll take sourcemaps thanks)

while firefox doesn't touch non-error objects
![image](https://github.com/user-attachments/assets/d4486e49-abe0-4d34-b4aa-c645ba58076f)

But it's also an issue when debugging node over `--inspect`, chrome tries it's best to find some links but only the Error object version is correct:

```
node --inspect-brk .\logger_test.js
```

![image](https://github.com/user-attachments/assets/bbe230b6-b078-4e97-9b40-61186cda55b9)


(at this point it's also worth considering if it's better to change issue requirements to just change into .warn(message, Error) as [diag api allows 2nd+ argument to be unknown](https://github.com/open-telemetry/opentelemetry-js/blob/91b9abdf210b58f9250ce443163c97faab4d7fd0/api/src/diag/types.ts#L20))